starwars/src/components/Character.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
+import './Character.css'
 
 const Character = (props) => {
     return(
         <div data-id={props.obj.created}>
             <h1>{props.obj.name}</h1>
-            <div className='card' onClick={props.bgHandler}>
+            <div className='card'>
```

Comment:
great work, could you explain the workflow of removing / moving your handler here?